docs/developers/guides/run-linea/run-the-stack/index.mdx

```diff
@@ -0,0 +1,31 @@
+---
+description: Run the Linea stack.
+image: /img/socialCards/run-the-linea-stack.jpg
+---
+
+import DocCardList from "@theme/DocCardList";
+
+# Run the Linea stack
+
+:::info
+
+The Linea stack is currently available only for local testing. This includes a local version of the
+base layer (Hyperledger Besu) and the Linea stack itself.
+ 
+:::
+
+Run the the entire Linea stack to get working versions of:
+
+- **Sequencer**: Responsible for ordering, building, and executing blocks.
+- **State manager**: Manages the L2 network and account states.
+- **Coordinator**: Coordinates the steps to create zk proofs and ensures their persistence on the
+    L1 Ethereum network. It specifically handles batch conflation, blob creation, and aggregation.
+- **Traces API**: Provides trace counts and generates conflated trace files for zk proof creation.
+- **Provers**: Generates three types of zk-SNARK proofs: execution, compression, and aggregation, it dynamically
+    adjusts its instances based on the volume of proof requests.
```

Comment:
The dynamic instance adjustments isn't a behaviour of the local stack, this is a production feature only, with the local stack it is only a single instance of the prover.